src/test/java/jssc/SerialNativeInterfaceTest.java

```diff
@@ -1,16 +1,24 @@
 package jssc;
 
 import org.junit.Assume;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 public class SerialNativeInterfaceTest {
 
+    private SerialNativeInterface testTarget;
+
+    @Before
+    public void before(){
+        testTarget = new SerialNativeInterface();
+    }
```

Comment:
This should best be moved from `@Before` directly to `throwsIllegalArgumentExceptionIfPortHandleIllegal()` method - just as it's done in `reportsWriteErrorsAsIOException`. We have no need to rerun this block for each test in this class, if only one is ever going to make use of it.